### ISSUE TITLE
Openhab binding cooperation 

### ISSUE THREAD

**Issue by Supermagnum** [rcastberg/sure_petcare#10]
There is a discussion here that is about the integration of  the python library for accessing sure connect petflap in to Openhab. The open Home Automation Bus (openHAB, pronounced ˈəʊpənˈhæb) is an open source, technology agnostic home automation platform.
I suggest that the programmer of the library cooperates with the openHAB community so a complete public binding can be made. 
https://community.openhab.org/t/request-sureflap-pet-door-connect-binding/50417/11

**Comment by benleb** [comment]
I think we should never implement openHab/Home assistant specific stuff in this lib. Any such stuff should be own project/component/binding/whatever.

**Comment by benleb** [comment]
I think we should never implement openHab/Home assistant specific stuff in this lib. Any such stuff should be own project/component/binding/whatever.

**Comment by rcastberg** [comment]
I completely agree with @benleb answer. OpenHab are welcome to use the library and implement a component.
Personally I will soon remove the home assistant component from this repository and add it to its own component, which will pull from this repo/pip.

**Comment by strix-technica** [comment]
Yeah, maintaining the API is enough work on its own when random stuff changes.  This is the wrong place for integrations because they can't/won't get tested.  (I don't (yet) have even a hass instance to test with, for example.)
